app/services/flow/flow_state_machine.rb

```diff
@@ -124,7 +124,8 @@ def call_optional_show_step(step)
       result = optional_show_step.new(@flow).base_call
 
       if @analytics_id
-        optional_properties = result.to_h.merge(step: optional_show_step)
+        optional_show_step_name = optional_show_step.to_s.split('::').last.underscore
```

Comment:
Ooh, a chance to use an arcane rails helper I learned recently: [#demodulize](https://apidock.com/rails/String/demodulize)